contrib/drivers/dutlink/jumpstarter_driver_dutlink/driver.py

```diff
@@ -82,6 +82,22 @@ def control(self, direction, ty, actions, action, value):
             return str_value
 
 
+@dataclass(kw_only=True)
+class DutlinkSerial(DutlinkConfig, PySerial):
+    url: str | None = field(init=False, default=None)
+    alternate_console: str | None = field(default=None)
```

Comment:
```suggestion
    alternate_console: str | None = field(default=None)
```

I think in this case the alternate console is not necessary anymore. If you are going to use an alternate console standalone, you should just use pyserial.

The alternate console is for some devices that will expose the DUT console as a USB port that comes from the DUT itself. I would just open the console found for the specific serial number, and that's it.

TL;DR

i.e. you can see special handling here https://github.com/jumpstarter-dev/jumpstarter/pull/223/files#diff-a1e61e9045a78ae1ee5d8ac8863402c4dfb5237c04fa246119da0c3fe3df8e64R269 because in some cases the DUT has to be powered on for the console to show up, if anybody needs that type of handling should go for the composite DUTLink and not care about this.